hub-remote-deploy

```diff
@@ -0,0 +1,196 @@
+#!/bin/bash -e
+
+usage() {
+    cat << EOF
+
+Executes deployment of Certified Stack from kuberentes cluster
+
+Usage: 
+$ $(basename "$0") -c component1,component2
+Deploys deploys two components 
+
+Parameters:
+    -c --components       List of components to deploy
+    -o --offset           Deploy starting with the given component
+    -s  --silent          Suppress console outputs in favor of result codes
+    -p --no-precheck      Skip `hub ext aws status` command execution
+    -d --delete-toolbox   Deletes deployed toolbox 
+    -k --keep-toolbox     Keeps toolbox deployed on kubernetes cluster
+    --skip-guide          Suppress "What's next messages"
+    -V  --verbose         Verbose outputs for debug purpose
+    -h  --help            Print this message
+
+EOF
+}
+
+generate_manifests() {
+    trap "rm -rf $KUSTOMIZE_DIR" EXIT
+
+    envfile="$KUSTOMIZE_DIR/envfile"
+    env | grep -E '^(AWS_|GOOGLE_|AZURE_|TF_|TERM=|LANG=|LC_)' >"$envfile"
+    kustomize_manifest="$KUSTOMIZE_DIR/kustomization.yaml"
+
+    cat <<EOF > $kustomize_manifest
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+commonLabels:
+  app: toolbox
+  provider: agilestacks.com
+configMapGenerator:
+- name: toolbox-env
```

Comment:
There may be a collision... If I want to run several toolboxes simulteniously

Reply:
CM won't be an issues(they have random postfix), but toolbox pod it self will have an issue.  I will fix that, but I'll have to store container uniq id locally.